.github/workflows/turborepo-release.yml

```diff
@@ -217,7 +217,7 @@ jobs:
           - host: ubuntu-latest
             container: ubuntu:xenial
             container-options: "--platform=linux/amd64 --rm"
-            container-setup: "apt-get update && apt-get install -y curl musl-tools"
+            container-setup: "apt-get update && apt-get install -y curl musl-tools sudo"
```

Comment:
Required because the capnproto action needs `sudo`, which isn't installed in the `linux/amd64 xenial` container. We could remove the sudo from the capnproto action, but because it's used for all linux distros this was the easier option.